i18n/qqq.json

```diff
@@ -24,9 +24,11 @@
 	"documentation": "Link text in the footer for the documentation link\n{{identical|Documentation}}",
 	"version": "Link text in the footer for the source code link.\n\nParameter:\n* $1 – the current application version number.\n\n{{identical|Version}}",
 	"report-issue": "Link text in the footer for the issue-reporting link.",
+	"langs-placeholder": "Placeholder text for the language input field.",
 	"langs-param-error": "Error message displayed when invalid language(s) are submitted.\n\nParameters:\n* $1 – number of invalid languages\n* $2 - the list of invalid languages",
 	"tesseract-options": "Heading for Tesseract-specific options.",
 	"tesseract-psm-label": "Form label for the Tesseract page segmentation mode.",
+	"tesseract-psm-help": "Help text for the Tesseract page segmentation mode option. 'Sparse text' refers to options, see messages {{msg-wm|tesseract-psm-11}} and {{msg-wm|tesseract-psm-12}}.",
```

Comment:
The way I've seen it done on many MW repos is

  {{doc-important|Do not translate \"Sparse text\".}}

can we use it here?

Reply:
We _do_ want them to translate it, but it should match the wording used in the 'tesseract-psm-11' and 'tesseract-psm-12' messages. I probably could have worded this better

Reply:
Oh, I'm sorry, I didn't get it.